convert2rhel/main.py

```diff
@@ -363,3 +355,31 @@ def rollback_changes():
             loggerinst.info("During rollback there were no backups to restore")
         else:
             raise
+
+    return
+
+
+def print_info_after_rollback(pre_conversion_results, include_all_reports):
+    """Print after-rollback messages. Also determine if there is a report to print or
+    report should't be printed after failure in rollback."""
```

Comment:
I don't like the name too much as we not only do logging in it but also create a report summary. Not sure of the best name tbh

Reply:
Renamed to `provide_status_after_rollback`